libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -65,7 +65,8 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Param: AUTO_SWITCH
     // @DisplayName: Automatic Switchover Setting
     // @Description: Automatic switchover to GPS reporting best lock
-    // @Values: 0:Disabled,1:Enabled
+    // @Values: 0:Disabled,1:Hard,2:Soft
+    // @RebootRequired: True
     // @User: Advanced
     AP_GROUPINFO("AUTO_SWITCH", 3, AP_GPS, _auto_switch, 1),
```

Comment:
True, but after this is ready for master, I'm more confident in this method than the previous one but maybe that's just me. I'm fine with putting it to disabled by default too - it's just leaving it at the old method by default doesn't look good to me.